Application/FileConverter/Settings.cs

```diff
@@ -207,6 +208,20 @@ public bool CheckUpgradeAtStartup
             }
         }
 
+        [XmlElement]
+        public bool CheckCopyFilesAfterConverting
+        {
+            get
+            {
+                return this.checkCopyFileAfterConverting;
+            }
+
+            set
+            {
+                this.checkCopyFileAfterConverting = value;
+                this.RaisePropertyChanged();
```

Comment:
I made some changes removing deprecated dependencies. You can now use NotifyPropertyChanged instead of RaisePropertyChanged.